lib/gateway/Shard.js

```diff
@@ -578,12 +579,12 @@ class Shard extends EventEmitter {
                 * @prop {Object} emoji The reaction emoji object
                 * @prop {String?} emoji.id The emoji ID (null for non-custom emojis)
                 * @prop {String} emoji.name The emoji name
-                * @prop {String} userID The ID of the user that added the reaction
+                * @prop {Member | String} reactor The member if in a guild or the ID of the user that added the reaction
                 */
                 this.emit("messageReactionAdd", message || {
                     id: packet.d.message_id,
                     channel: channel || {id: packet.d.channel_id}
-                }, packet.d.emoji, packet.d.user_id);
+                }, packet.d.emoji, member || packet.d.user_id);
```

Comment:
Copy 586 instead of `| String`

Reply:
I'm not sure what you meant by this abal. Can you please clarify what you need to have this merged